packages/app-cli/tests/services_SearchEngine.js

```diff
@@ -511,4 +511,14 @@ describe('services_SearchEngine', function() {
 		expect((await engine.search('"- [ ]"', { searchType: SearchEngine.SEARCH_TYPE_BASIC })).length).toBe(1);
 		expect((await engine.search('"[ ]"', { searchType: SearchEngine.SEARCH_TYPE_BASIC })).length).toBe(2);
 	}));
+
+	it('should not mistake cyrillic "l" for latin "n"', (async () => {
+		const n1 = await Note.save({ title: 'latin n', body: 'n' });
+		const n2 = await Note.save({ title: 'cyrillic l', body: 'л' });
+
+		await engine.syncTables();
+
+		expect((await engine.search('n'))[0].id).toBe(n1.id);
+		expect((await engine.search('л'))[0].id).toBe(n2.id);
```

Comment:
Thanks for adding this, but could you also put back the check on the number of results please?

Reply:
done